monitoring/podMonitorGuide.md

```diff
@@ -0,0 +1,96 @@
+# PodMonitor
+
+When using Dragonfly operator, sometimes there may be a need to monitor the dragonfly instances. PodMonitors can be of great use here. In this doc, we will see how we can configure PodMonitor resource to monitor dragonfly instances.
+
+## Step 1:
+First make sure you have all the prometheus crds installed in your cluster. Here we are going to install prometheus operator.
+
+```
+LATEST=$(curl -s https://api.github.com/repos/prometheus-operator/prometheus-operator/releases/latest | jq -cr .tag_name)
+curl -sL https://github.com/prometheus-operator/prometheus-operator/releases/download/${LATEST}/bundle.yaml | kubectl create -f -
+```
+
+## Step 2:
+Now that we have installed the operator, we can create `prometheus` resources. If you have RBAC enabled, create necessary `serviceaccount`, `clusterrole` and `clusterrolebinding` resources first.
+
+ServiceAccount -
```

Comment:
Can you switch to commands here directly? to make it more usable and tryable? like we have in the readme?

Reply:
Ok sure